### PR TITLE
Add 1.8.2 notes

### PR DESCRIPTION
**Pull request by miekg**
Signed-off-by: Miek Gieben <miek@miek.nl>

### CODE REVIEW

**Comment by miekg** [comment]
also fixed a zsh-ism in checking for the curl command

File: Makefile.release [hunk 1]
```diff
@@ -44,7 +44,7 @@ EMPTY:=
 SPACE:=$(EMPTY) $(EMPTY)
 COMMA:=$(EMPTY),$(EMPTY)
 
-ifeq (, $(shell command -v curl))
```
Comment (mrueg) [inline Makefile.release:47]: Why moving back to which? `command -v` is posix https://pubs.opengroup.org/onlinepubs/009604499/utilities/command.html https://github.com/koalaman/shellcheck/wiki/SC2230

Comment (yongtang) [reply Makefile.release:47]: On Ubuntu 20.04 with `command` I noticed the following errors:
```
$ make -f Makefile.release  tar
make: command: Command not found
Makefile.release:48: *** "No curl in $PATH, please install".  Stop.
```

**Comment by yongtang** [comment]
@miekg `which -v` may not work on Ubuntu 20.04 as well:
```
$ which -v curl
Illegal option -v
Usage: /usr/bin/which [-a] args
```

Maybe we can simply go with `which curl`?

File: Makefile.release (hunk 1 above)
Comment (mrueg) [reply Makefile.release:47]: Okay, I don't have a ubuntu machine to test here. Then yes revert back to `which curl` as @yongtang , I was hoping that we could avoid relying on which.

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] Add 1.8.2 not..." ]
>@miekg which -v may not work on Ubuntu 20.04 as well:
>
>$ which -v curl
>Illegal option -v
>Usage: /usr/bin/which [-a] args
>
>Maybe we can simply go with which curl?

yes, that should be enough, and was what was there for the manifest-tool

**Review by yongtang — APPROVED** [review]
LGTM

**Pull request merged**